File: Groove.SP/Groove.SP.Core/Entities/ShippingComplianceModel.cs
using Groove.SP.Core.Models;

namespace Groove.SP.Core.Entities
{
    public class ShippingComplianceModel : Entity
    {
        public long Id { get; set; }

        public long BuyerComplianceId { get; set; }

        public BuyerComplianceModel BuyerCompliance { get; set; }

        public bool AllowPartialShipment { get; set; }

        public bool AllowMixedCarton { get; set; }

        public AllowMixedPackType AllowMixedPack { get; set; }

        public bool AllowMultiplePOPerFulfillment { get; set; }

    }
}

File: Groove.SP/groove-sp-web/src/app/features/compliance/models/shipping-compliance.model.ts
import { AllowMixedPack } from 'src/app/core';

export class ShippingComplianceModel {
    id: number;
    allowPartialShipment: boolean;
    allowMixedCarton: boolean;
    allowMixedPack: AllowMixedPack;
    allowMultiplePOPerFulfillment: boolean;
}